gradle/libs.versions.toml

```diff
@@ -2,12 +2,12 @@
 [versions]
 accompanistPager = "0.35.1-alpha"
 activityCompose = "1.9.0"
-agp = "8.4.1"
-androidTools = "31.4.2"
+agp = "8.5.0"
```

Comment:
💪 

Note for everyone: @vishnuravi @Basler182, please make sure to update your android studio to Koala after this PR will get merged (see [Android Gradle plugin and Android Studio compatibility](https://developer.android.com/build/releases/gradle-plugin#android_gradle_plugin_and_android_studio_compatibility))